### PR TITLE
fixed overlay with color name of the gradient in chart sidebar 

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
fixed overlay with color name of the gradient in chart sidebar 
Change-Id: Ie3c98a908efedfd666cbafa22f21f907f947eb7a

* Resolves: #6293 
* Target version: master 

@pedropintosilva , @GulsahKose

### CODE REVIEW

File: browser/css/jssidebar.css [hunk 1]
```diff
@@ -130,12 +130,12 @@ div.sidebar.ui-grid .checkbutton.sidebar,
 	position: absolute;
 	display: block;
 	width: auto;
-	background-color: var(--color-background-darker);
+	background-color: transparent;
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:133]: I think we should make sure the hover status for the menubutton has the same border color and background color as other components such as.jsdialog.ui-listbox:hover 

also `.menubutton.sidebar:hover span` should be improved  ideally to look like any other regular `.ui-tooltip`

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
let me push one commit before you do any further changes

**Comment by pedropintosilva** [comment]
Thanks @Darshan-upadhyay1110 but there were other fixes that were needed before attempting to do this one, and some other PRs that changes some CSS values. So I ended up pushing https://github.com/CollaboraOnline/online/pull/6413/commits 

Could you please test that ^ ? I see there are now a regression WRT width of those components (but it is something unrelated and new introduced by a new width rule that is applied in the overall btn styles so it is not a blocker for my PR)

**Comment by Darshan-upadhyay1110** [comment]
okay sure i will test that

**Pull request closed without merging**